src/sentry_clj/ring.clj

```diff
@@ -1,9 +1,14 @@
 (ns sentry-clj.ring
   "Ring utility functions."
   (:require
+   [clojure.string :refer [upper-case]]
    [ring.util.request :refer [request-url]]
    [ring.util.response :as response]
-   [sentry-clj.core :as sentry]))
+   [sentry-clj.core :as sentry]
+   [sentry-clj.tracing :as st])
+  (:import
+   (io.sentry SentryEvent EventProcessor)
```

Comment:
Can these `imports` be changed to vectors please, as in in `core`?

Reply:
OK. I'll use vectors, not list.